version/version.go

```diff
@@ -10,3 +10,10 @@ var (
 	// Revision is filled with the VCS revision. Filled in at linking time.
 	Revision = ""
 )
+
+func FullVersion() string {
```

Comment:
Missing documentation for this function.

Reply:
added.